apps/frontend/src/components/global/groups/Users.vue

```diff
@@ -90,14 +93,24 @@ export default class Users extends Vue {
     {
       text: 'Role',
       value: 'groupRole'
-    },
-    {
-      text: 'Actions',
-      value: 'actions',
-      sortable: false
     }
   ];
 
+  get displayedHeaders() {
+    // If the user is editing the group, then display the actions column.
+    if(this.editable) {
+      this.headers.push({
+        text: 'Actions',
+        value: 'actions',
+        sortable: false,
+      });
+      return this.headers;
+    }
+    else {
+      return this.headers;
+    }
```

Comment:
This is redundant, you can just always `return this.headers` outside of the if/else statement.